java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -992,6 +1081,23 @@ public final ColumnBuilder append(boolean value) {
       return this;
     }
 
+    public final ColumnBuilder append(BigDecimal value) {
```

Comment:
This should be implemented using the `append(BigDecimal, RoundingMode)` method.

Reply:
This method is only for decimal type contained by nested types. There is no place to pass `RoundingMode` in builder of nested type. So, I fixed the RoundingMode on `UNNECESSARY`.